app/controllers/app_configs_controller.rb

```diff
@@ -35,6 +35,22 @@ def update
     end
   end
 
+  def run_daily_tasks
+    if can? :access, :rails_admin
```

Comment:
We can use a new permission here (e.g. `can? :run, :jobs`), since superusers have effectively `sudo`-like access so they can satisfy anything. I think we should similarly blacklist admins from using these buttons, for now, but by decoupling the permissions we can choose to change things later.